vtex/faststore
# Chore: Copy stores public folder to .faststore

## What's the purpose of this pull request?

As reported in #2020 , today we can't customize the `robots.txt` file. This PR enables our CLI to copy the public folder from the store to `.faststore`.

Today we copy the public folder from `.faststore` to Core when we build because of our CICD. I made a change in our CICD so we don't need that anymore.

CICD PR: https://github.com/vtex/vtex-cicd-platform/pull/520

## How to test it?

This PR overwrites the public folder: https://github.com/vtex-sites/starter.store/pull/236. You can change the files and check the `.faststore` or check those preview URLs:

`test.js`: File not allowed - https://sfj-a7ded08--starter.preview.vtex.app/test.js
`test.json`: File allowed - https://sfj-a7ded08--starter.preview.vtex.app/test.json
`robots.txt`: File allowed - https://sfj-a7ded08--starter.preview.vtex.app/robots.txt

### Starters Deploy Preview

https://sfj-a7ded08--starter.preview.vtex.app/

## References
